packages/legend-query-builder/src/stores/QueryBuilderResultState.ts

```diff
@@ -417,7 +418,17 @@ export class QueryBuilderResultState {
               this.queryBuilderState.graphManagerState,
             ),
           },
+          undefined,
+          contentType,
         )) as Response;
+      if (result.headers.get(V1_DELEGATED_EXPORT_HEADER) === 'true') {
+        if (result.status === 200) {
+          this.exportState.pass();
```

Comment:
how will the users get notified here. is the delegated client in charge of notifying that the download will run in the background ?

Reply:
The user will still get the `Export CSV will run in background` notification within query builder (i.e., the VS Code webview). The delegated client (i.e., vs code) will then handle actually saving the file and displaying a message if the download succeeds/fails.